### PR TITLE
Add script to copy metadata to test reporting

### PR DESCRIPTION
**Pull request by vladsavelyev**

### CODE REVIEW

**Comment by lgruen** [comment]
I believe this is obsolete with the new `metadata` buckets.

**Comment by vladsavelyev** [comment]
Wanted to update it to copy the joint-calling meta.tsv file.

File: scripts/data_transfer/copy_test_data_for_reports.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+is symmetrical to gs://cpg-tob-wgs-main/joint-calling/v2/meta.tsv and limited to samples in the
+test subset.
+"""
+
+import os
+import subprocess
+import click
+
+output = os.getenv('OUTPUT')
+assert output and output.startswith('gs://cpg-tob-wgs-test/')
```
Comment (lgruen) [inline scripts/data_transfer/copy_test_data_for_reports.py]:
```suggestion
assert output and output.startswith('gs://cpg-tob-wgs-main-metadata/')
```

File: scripts/data_transfer/copy_test_data_for_reports.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+@click.option('--version', help='Joint-calling run version', required=True)
+def main(version):
+    """Entry point"""
+
+    subprocess.run(
+        [
+            'gsutil',
+            'cp',
+            f'gs://cpg-tob-wgs-main/joint-calling/{version}/meta.tsv',
+            f'gs://cpg-tob-wgs-main-metadata/joint-calling/{version}/meta.tsv',
```
Comment (lgruen) [inline scripts/data_transfer/copy_test_data_for_reports.py]: Did you mean to use `output` here?

File: scripts/data_transfer/copy_test_data_for_reports.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+output = os.getenv('OUTPUT')
+assert output and output.startswith('gs://cpg-tob-wgs-test/')
+
+
+@click.command()
+@click.option('--version', help='Joint-calling run version', required=True)
+def main(version):
+    """Entry point"""
+
+    subprocess.run(
```
Comment (lgruen) [inline scripts/data_transfer/copy_test_data_for_reports.py]: Note that you can now also run simple bash scripts in the analysis-runner.

File: scripts/data_transfer/copy_test_data_for_reports.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
+"""
+Copy a sample-QC metadata TSV file generated by the joint-calling pipeline into the test bucket.
+The idea is that for interactive exploration of QC summary metrics and design of plots, it's safe
+to allow using the full set of samples, because the metadata doesn't contain any actual genomics
+variantion data. 
+
+Note that the test bucket also contains gs://cpg-tob-wgs-test/joint-calling/v2/meta.tsv, which
```
Comment (lgruen) [inline scripts/data_transfer/copy_test_data_for_reports.py]: Should probably be updated to refer to `test-metadata` now.

**Review by lgruen — CHANGES_REQUESTED** [review]

File: scripts/data_transfer/copy_test_data_for_reports.py (hunk 3 above)
Comment (vladsavelyev) [reply scripts/data_transfer/copy_test_data_for_reports.py]: Good point, a bash script would better suit here

File: scripts/data_transfer/copy_test_data_for_reports.py (hunk 2 above)
Comment (vladsavelyev) [reply scripts/data_transfer/copy_test_data_for_reports.py]: But since we don't want the destination to be configurable, why use a variable here?

**Comment by vladsavelyev** [comment]
Replaced with a bash script :)

File: scripts/data_transfer/copy_test_data_for_reports.py (hunk 2 above)
Comment (lgruen) [reply scripts/data_transfer/copy_test_data_for_reports.py]: Well, we also copy a metadata file to `output` in the analysis-runner that describes that particular run. If you're not actually writing to that location, these destinations can be out of sync.

File: scripts/data_transfer/copy_joint_calling_metadata.sh [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+VERSION=$1
+
+if [[ -z "$VERSION" ]]; then
+    echo "Provide a version tag of a joint-calling run as a first argument" 1>&2
+    exit 1
+fi
+
+gsutil cp \
+	gs://cpg-tob-wgs-main/joint-calling/${VERSION}/meta.tsv \
+	gs://cpg-tob-wgs-main-metadata/joint-calling/${VERSION}/meta.tsv
```
Comment (lgruen) [inline scripts/data_transfer/copy_joint_calling_metadata.sh]: Please see my other comment about potentially still using `$OUTPUT` here.

**Review by lgruen — APPROVED** [review]

**Pull request merged**